torch/tensor.py

```diff
@@ -528,7 +534,7 @@ def __format__(self, format_spec):
             return self.item().__format__(format_spec)
         return object.__format__(self, format_spec)
 
-    def __ipow__(self, other):
+    def __ipow__(self, other):  # type: ignore[misc]
```

Comment:
this is probably the only issue I dont know what's the right way to fix:

apparently `__ipow__` is not defined in the generated `_C/__init__.pyi` and in previous line `__pow__ = _C._TensorBase.pow` instead of `_C._TensorBase.__pow__` (which is defined BTW). it got me a bit confused.

Reply:
I mean, reading the code here, it sounds like we don't implement `__ipow__` (that's why we raise `NotImplemented` here). So it isn't unexpected that it's not defined in `_C`. What's the problem?

Reply:
oops. forgot to mention the problem - mypy expects identical signature between `__pow__` and `__ipow__` since `a **= b` is equivalent to `a = a ** b`. Yes I think we are expected to not implemented this function so I will just leave it as ignore for now.